pyfstat/make_sfts.py

```diff
@@ -154,16 +155,20 @@ def _get_setup_from_tstart_duration(self):
         self.duration = int(self.duration)
 
         IFOs = self.detectors.split(",")
-        numSFTs = len(IFOs) * [int(float(self.duration) / self.Tsft)]
+        # when duration is not a multiple of Tsft, to match MFDv5
+        # we need to round the number *up*
+        # and also include the overlapping bit at the end in the duration
+        numSFTs = int(np.ceil(float(self.duration) / self.Tsft))  # per IFO
+        effective_duration = numSFTs * self.Tsft
```

Comment:
I'd say go for XLALMakeTimestamps .

Reply:
WIll make a separate issue for that; we don't support Toverlap so far so the only thing we would gain for now would be to be consistent with lal's fudging in the ceil call, which should only make a difference in rare corner cases.